contracts/vaults/PerpLPVault.sol

```diff
@@ -271,10 +314,9 @@ contract PerpLPVault is IPerpLPVault, IntegrationVault {
         );
     }
 
-    function _isStrategy(address addr) internal view returns (bool) {
-        return _vaultGovernance.internalParams().registry.getApproved(_nft) == addr;
-    }
-
+    /// @notice Close a Uni position, if the position has non-zero funds
```

Comment:
I'd explain how a permanent position emerges (see https://support.perp.com/hc/en-us/articles/5257492022809 - section Impermanent to Permanent)